extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -338,6 +318,11 @@ export class AmpA4A extends AMP.BaseElement {
      *    of resume callback.
      */
     this.fromResumeCallback = false;
+
+    const signatureVerifier = signatureVerifierFor(this.win);
+    this.getSigningServiceNames().forEach(signingServiceName => {
```

Comment:
I realize we currently fetch public keys in the constructor but should this be done instead in buildCallback?  Should it be blocked by not in prerender state?